daemon/mgr/container.go

```diff
@@ -50,7 +50,10 @@ import (
 type ContainerMgr interface {
 	// 1. the following functions are related to regular container management
 
-	// Restore containers from meta store to memory and recover those container.
+	// Register containers from meta store to memory.
+	Register(ctx context.Context) error
```

Comment:
IMO, named `Register` is not suitable, how about `Load`?

Reply:
make sense